src/Avalonia.Controls/Primitives/SelectingItemsControl.cs

```diff
@@ -551,6 +558,7 @@ bool Match(Control container)
             base.OnTextInput(e);
         }
 
+        /// <inheritdoc />
```

Comment:
XML docs for couple of years is now automatically inherited. inheritdoc is not necessary here.

Reply:
I thought it still triggered the compiler warning about missing the XML documentation (as a suggestion here). I can revert those if you prefer, I added it to all methods which had inherited doc on files I modified, might have been a little carried away here :)

Reply:
Needs to be double checked if we have these warnings by not having inheritdoc here.
Good question if we should add inheritdoc then, or disable warning and report to the https://github.com/dotnet/roslyn-analyzers ...

Reply:
I double checked: `inheritdoc` is definitely needed to prevent the warning (or the suggestion here since the severity has been changed in `.editorconfig`.)

https://github.com/dotnet/roslyn/issues/65386#issuecomment-1312324309 confirms that documentation comments aren't automatically inherited (they don't appear in the resulting XML file without `inheritdoc`): it's an IDE-time feature for convenience.